app/src/main/java/fr/free/nrw/commons/nearby/PlaceRenderer.java

```diff
@@ -44,8 +44,10 @@ public void render() {
         Place place = getContent();
         tvName.setText(place.name);
         String descriptionText = place.getLongDescription();
+        tvDesc.setVisibility(View.VISIBLE);
         if (descriptionText.equals("?")) {
             descriptionText = getContext().getString(R.string.no_description_found);
+            tvDesc.setVisibility(View.INVISIBLE);
         }
```

Comment:
There was an similar PR you can see here: #1072 . As you can read the explanation there, you need to add an else statement as psh noted.

Reply:
@neslihanturan  it won't be needed because just before if I am explicitly setting the visibility for each item in recycler view. So it would be invisible in case of No Description only.

Reply:
Oh, I just recognized line 47. Yes there is no problem then